.github/workflows/build.yaml

```diff
@@ -201,37 +205,55 @@ jobs:
 
 
   publishdocker:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Any reason for 20.04... ?

Might be worth doing everything on the same docker version... latest or :2004

Reply:
100% agree on the consistency stake.  

Enyeart made a convincing argument over in the fabric repos that we should be pinning the base build revs.  Let's try ubuntu-20.04 to align with the fabric builds.